extensions/moderation/timeout.py

```diff
@@ -99,8 +104,10 @@ async def timeout(
             except discord.HTTPException or discord.Forbidden:
                 pass
             setting = await self.client.db.get_setting(setting=SettingsEnum.ModLogChannel, guild=ctx.guild)
-            log_channel: discord.TextChannel = await get_or_fetch(ctx.guild, "channel", setting, default=None)
-            await log_channel.send(embed=member_em)
+            if setting:
+                log_channel: discord.TextChannel = await get_or_fetch(ctx.guild, "channel", setting.value, default=None)
+                if log_channel:
+                    await log_channel.send(embed=member_em)
         except discord.Forbidden or discord.HTTPException as e:
             raise e
 
```

Comment:
ignore complex method, can't break it more down while keeping it usable/readable